kernel/Kconfig

```diff
@@ -983,6 +983,29 @@ config KERNEL_WHOLE_ARCHIVE
 	  This option forces every object file in the libkernel.a archive
 	  to be included, rather than searching the archive for required object files.
 
+config STATIC_INIT_GNU_SUPPORTED
```

Comment:
Given that other toolchain feature configs are named like `TOOLCHAIN_SUPPORTS_x` (e.g. see `TOOLCHAIN_SUPPORTS_THREAD_LOCAL_STORAGE` above), it would be better to name this `TOOLCHAIN_SUPPORTS_STATIC_INIT_GNU`.

Reply:
changed